longevity_test.py

```diff
@@ -37,13 +38,18 @@ def test_custom_time(self):
                                     loaders=self.loaders,
                                     monitoring_set=self.monitors)
         stress_queue = list()
-        for cmd in ('stress_cmd', 'stress_cmd_1'):
-            stress_cmd = self.params.get(cmd)
-            if stress_cmd:
-                if 'counter_' in stress_cmd:
-                    self._create_counter_table()
-                self.log.debug('stress cmd: {}'.format(stress_cmd))
-                stress_queue.append(self.run_stress_thread(stress_cmd=stress_cmd))
+        for stress_cmd in self.params.get('stress_cmd'):
```

Comment:
It doesn't support original type (single str).
Let's check if it's a list, then process two kinds of case.